<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWindow</class>
 <widget class="QDialog" name="StatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Статистика теста</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 14pt; font-weight: bold; color: #2c3e50; padding: 10px;</string>
     </property>
     <property name="text">
      <string>Статистика теста: [Название теста]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="infoLayout">
     <item>
      <widget class="QLabel" name="attemptsLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 11pt; color: #7f8c8d;</string>
       </property>
       <property name="text">
        <string>Всего попыток: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="averageLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 11pt; color: #7f8c8d;</string>
       </property>
       <property name="text">
        <string>Средний балл: 0.0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resultsTable">
     <property name="styleSheet">
      <string notr="true">
QTableWidget {
    background-color: white;
    alternate-background-color: #f8f9fa;
    gridline-color: #dee2e6;
    border: 1px solid #dee2e6;
    border-radius: 4px;
}

QTableWidget::item {
    padding: 8px;
    border-bottom: 1px solid #dee2e6;
    color: black;
}

QTableWidget::item:alternate {
    background-color: #f8f9fa;
}

QTableWidget::item:selected {
    background-color: #3498db;
    color: white;
}

QHeaderView::section {
    background-color: #34495e;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Ученик</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Баллы</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Результат</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата и время</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 8px 20px;
    background-color: #95a5a6;
    color: white;
    border: none;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #7f8c8d;
}</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>StatsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>